src/components/cylc/Toolbar.vue

```diff
@@ -32,7 +32,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <!-- TODO: duplicated in workflow/Toolbar.vue and cylc/Toolbar.vue -->
       <!-- burger button for mobile -->
       <v-btn
-        dark
+        light
```

Comment:
It looks like the default is `light` anyway. If we hard code it it might introduce the opposite bug when we implement dark mode #336
```suggestion
```